app/views/offline_reservations/new.html.haml

```diff
@@ -0,0 +1,19 @@
+= content_for :h1 do
+  = current_facility
+
+= content_for :sidebar do
+  = render "admin/shared/sidenav_product", sidenav_tab: "instruments"
+
+= content_for :tabnav do
+  = render "admin/shared/tabnav_product", secondary_tab: "reservations"
+
+%h2= @instrument
+
+= simple_form_for([@instrument, @reservation], url: facility_instrument_offline_reservations_path) do |f|
```

Comment:
Seems like there's a lot that could be shared between new and edit. I think the only difference is the text on the submit button.